resourcegraph.tf

```diff
@@ -0,0 +1,77 @@
+resource "azurerm_automation_module" "az_accounts" {
```

Comment:
What happens, when the modules are already installed by some other means? Is it 

1. reasonable to have this as prerequisite (dependency inversion)
2. at least output the installed modules so others can use it
3. Ignore, as we don't got the problem right now

Reply:
@QBY-MarkusMaring

Reply:
Current solution is Nr.3. The only reason we need to install these modules is because there is an error with the default installed module versions (see the warning banner under any automation account -> modules). If that gets fixed we could drop this entirely.

If another module deploys the same pwsh module it will replace it in the AAC without an error (potentially updating it). This might break the previous runbook.